decidim-meetings/app/helpers/decidim/meetings/directory/application_helper.rb

```diff
@@ -109,7 +109,7 @@ def directory_filter_categories_values
         def directory_filter_origin_values
           origin_values = []
           origin_values << TreePoint.new("official", t("decidim.meetings.meetings.filters.origin_values.official"))
-          origin_values << TreePoint.new("citizens", t("decidim.meetings.meetings.filters.origin_values.citizens"))
+          origin_values << TreePoint.new("participants", t("decidim.meetings.meetings.filters.origin_values.participants"))
```

Comment:
Maybe you need to change or remove the key for other languages as well?

Reply:
Crowdin will take care of it automatically after this is merged in `develop`